weldx/transformations/local_cs.py

```diff
@@ -686,9 +687,9 @@ def _interp_time_coordinates(self, time: Time) -> xr.DataArray:
         if "time" not in self.coordinates.dims:  # don't interpolate static
             return self.coordinates
         if time.max() <= self.time.min():  # only use edge timestamp
-            return self.coordinates[0]
+            return self.coordinates.isel(time=0).data
```

Comment:
we could also skip the `.data` here but in that case we would have to decide which "edge" timestamp we want to apply, however this is now more consistent for coordinates and orientations